airflow/utils/log/log_reader.py

```diff
@@ -34,7 +34,7 @@ class TaskLogReader:
     """Task log reader"""
 
     def read_log_chunks(
-        self, ti: TaskInstance, try_number: int | None, metadata
+        self, ti: TaskInstance, try_number: int | None, metadata, log_type=None
```

Comment:
We should make the new argument keyword-only

Reply:
good call